tests/test_update_sensor.py

```
@@ -659,23 +742,23 @@ async def test_build_osm_url_returns_url(mock_hass, mock_config_entry, sensor):
-    aioclient_mock,
-    sensor,
-    cached,
-    payload,
-    expected_attr,
-    network_error,
-):
+    mock_hass: MagicMock,
+    mock_config_entry: MockConfigEntry,
+    aioclient_mock: AioClientMock,
+    sensor: MockSensor,
+    cached: bool,
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)